pkg/client/client.go

```diff
@@ -760,6 +762,10 @@ func (c *Client) CreateOrUpdateAPIService(apiService *apiregistrationv1beta1.API
 		return errors.Wrap(err, "retrieving APIService object failed")
 	}
 
+	apiService.ResourceVersion = oldAPIService.ResourceVersion
+	if len(oldAPIService.Spec.CABundle) > 0 {
```

Comment:
eventually we could try to build a patch using the api machinery pkg rather than manually